src/components/screens/ServiceTile.tsx

```diff
@@ -84,16 +94,35 @@ export const ServiceTile = ({
         <View style={!isVisible && styles.invisible}>
           {item.iconName ? (
             <Icon.NamedIcon
-              color={hasDiagonalGradientBackground ? colors.lightestText : undefined}
+              color={
+                iconStyle.color || hasDiagonalGradientBackground ? colors.lightestText : undefined
+              }
               name={item.iconName}
-              size={normalize(30)}
-              style={styles.serviceIcon}
+              size={iconStyle.size || normalize(30)}
+              strokeColor={iconStyle.strokeColor}
+              strokeWidth={iconStyle.strokeWidth}
+              style={[styles.serviceIcon, hasIconStyle && iconStyle]}
+            />
+          ) : item.svg ? (
+            <IconUrl
+              color={
+                iconStyle.color || hasDiagonalGradientBackground ? colors.lightestText : undefined
+              }
+              iconName={item.svg}
+              size={iconStyle.size || normalize(30)}
+              strokeColor={iconStyle.strokeColor}
+              strokeWidth={iconStyle.strokeWidth}
+              style={[styles.serviceIcon, hasIconStyle && iconStyle]}
             />
           ) : (
             <Image
               source={{ uri: item.icon || item.tile }}
               childrenContainerStyle={[
                 styles.serviceImage,
+                hasIconStyle &&
+                  !!item.icon && {
+                    height: iconStyle.size || normalize(30)
```

Comment:
the height was 40 before, why 30 now? info in commit message missing.

Reply:
We already had the `normalize(30)` value for `Icon.NamedIcon` before. So I also added `IconUrl` as `normalize(30)`. I made this change in the `Image` component because I thought it should be the same size as the icons.